### DISCUSSION TITLE
found a way to get bookmarklets when javascript is disabled

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#429] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
tell me if it is my Chromebook but if you copy the info of the bookmarklet then paste it in the bookmarks bar(don't press add yet) then go to the start and replace javascript with a run then hit submit then afterward go and change it back to javascript it should work. please tell me if it is my chromebook

**Comment by EmpoweredPhoenix** [comment]
replace javascript with run(word)

**Comment by rwg100** [comment]
nope doesnt work sadly

**Reply by EmpoweredPhoenix** [reply]
oh

**Reply by Aka-but-nice** [reply]
Send video

**Comment by Aka-but-nice** [comment]
If this works, you're officially smarter than Chrome devs
